src/applications/vaos/components/VideoVisitSection.jsx

```diff
@@ -35,6 +35,7 @@ export default function VideoVisitSection({ appointment }) {
               ? `description-join-link-${appointment.id}`
               : undefined
           }
+          aria-disabled={disableVideoLink ? 'true' : 'false'}
```

Comment:
I think it'd be better to have the false ternary equal `null`. Otherwise this looks 💯

Reply:
@1Copenut Why's that?

Reply:
@jbalboni Just my incomplete understanding. I did a quick Google search but didn't find the spec for `aria-disabled` first go round. Finding it this morning ( https://www.w3.org/WAI/PF/aria/states_and_properties#aria-disabled ) and "false" is a valid value. Please disregard my comment.